doc/pages/architecture/030-interface.md

```diff
@@ -87,7 +87,7 @@ Interfaces have a well-known, predefined type, which can be either `property` or
 persistent state or synchronization. As a rule of thumb, `datastream` interfaces should be used when
 dealing with values such as sensor samples, commands and events. `datastream` are stored as time
 series in the database, making them suitable for time span filtering and any other common time
-series operation, and they are not idempotent in the REST API semantics.
+series operation, but they cannot perform the same actions as they do in the REST API semantics.
```

Comment:
Personal opinion: I believe the previous statement about API idempotency was clearer and more correct.